pkg/pb/notifications.proto

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package notifications;
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+message Notification{
+    enum Scope {
+        CUSTOM = 0;
+        AUDIT = 1;
+        UPTIME = 2;
+        REPAIR = 3;
+        DISQUALIFICATION = 4;
+        GRACEFUL_EXIT = 5;
+        VETTING = 6;
+    }
+
+    enum Level {
+        INFO = 0;
+        WARN = 1;
+        ERROR = 2;
+    }
+
+    Scope scope = 1;
+    Level level = 2;
+    repeated string tags = 3;
+    string message = 4;
+    google.protobuf.Timestamp timestamp = 5 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+}
+
+message ReportRequest{
```

Comment:
Is there a difference between `ReportRequest` and a `Report`. Also `Report` isn't clear enough on what it contains.

Reply:
`Report` is a method name, `ReportRequest` is request that we pass